tensorflow/python/keras/layers/convolutional_test.py

```diff
@@ -730,35 +730,88 @@ def test_zero_padding_3d(self):
     input_len_dim2 = 5
     input_len_dim3 = 3
 
-    inputs = np.ones((num_samples, input_len_dim1, input_len_dim2,
-                      input_len_dim3, stack_size))
+    for data_format in ['channels_first', 'channels_last']:
```

Comment:
Rather than using `for` loops, prefer using a parameterized test such as this one: https://github.com/tensorflow/tensorflow/blob/f760e5bb88054d7957bbaec2329f9175f75769a6/tensorflow/python/keras/layers/convolutional_test.py#L67

Reply:
> Rather than using `for` loops, prefer using a parameterized test such as this one:
> 
> https://github.com/tensorflow/tensorflow/blob/f760e5bb88054d7957bbaec2329f9175f75769a6/tensorflow/python/keras/layers/convolutional_test.py#L67

Already update and pass the tests.